news-bundle/src/Resources/contao/classes/News.php

```diff
@@ -494,6 +494,37 @@ public function purgeOldFeeds()
 
 		return $arrFeeds;
 	}
+
+	/**
+	 * Change the page title to news title, if set.
+	 * @param PageModel   $objPage
+	 * @param LayoutModel $objLayout
+	 * @param PageRegular $objPageRegular
+	 */
+	public function setMetaData(PageModel $objPage, LayoutModel $objLayout, PageRegular $objPageRegular)
+    {
+        if (!\Input::get('items'))
+        {
+            return; // no news
+        }
+
+        $news = \NewsModel::findOneBy(array('alias=?', 'published=?'), array($this->Input->get('items'), 1));
```

Comment:
You can use `\Input::get('items')` here again, or even better, store it in a variable on line 506 :)